### ISSUE TITLE
drtmle errors when user inputs Qn and gn and returnModels = TRUE

### ISSUE THREAD

**Issue by benkeser** [benkeser/drtmle#16]

**Comment by jameszou1206** [comment]
I actually found a similar error if one specify guard = 'Q'.

**Comment by benkeser** [comment]
Thanks. Mind putting up a [reprex](https://github.com/tidyverse/reprex) and I can track down the bug?

**Comment by jameszou1206** [comment]
Never used reprex. But here's a simple example outputting this error:
```X <- runif(100, 0, 1)

Q <- X

g <- exp(X) / (1 + exp(X))

A <- rbinom(100, 1, g)

Y <- runif(Q, -0.1, 0.1)

X <- as.data.frame(X)

a <- drtmle(W = X, A = A, Y = Y, a_0 = 1, glm_Q = 'X', glm_g = 'X', SL_Qr = 'SL.npreg', guard = 'Q', returnModel = TRUE)

**Comment by benkeser** [comment]
How is `X` generated in your example?

**Comment by jameszou1206** [comment]
Sorry just edited my post.

**Comment by benkeser** [comment]
Thanks. Resolved by 9ec2bbb5abc8c2520344f156c7afa20274d45420
